tests/hawtio-test-suite/src/main/java/io/hawt/tests/features/pageobjects/pages/panel/help/HelpPage.java

```diff
@@ -10,7 +10,7 @@
 
 public class HelpPage extends HawtioPage {
     public void switchTab(String tab) {
-        $(byXpath("//div[@class='pf-v5-c-page__main-nav']//a[text()='" + tab + "']")).shouldBe(interactable).click();
+        $(byXpath("//nav[@aria-label='Nav']//a[text()='" + tab + "']")).shouldBe(interactable).click();
```

Comment:
Please consider using css selectors paired with ByUtils where methods for label and text selectors are located, they are more readable

Reply:
Ok, I will change it, thanks

Reply:
@mmuzikar can you check it now, please?